docs/documenting/create-page.md

```diff
@@ -2,9 +2,20 @@
 title: "Create a new page"
 ---
 
-For a platform or integration page, the fastest way is to make a copy of an existing page and edit it. The [Integration overview](https://www.home-assistant.io/integrations/) and the [Examples section](https://www.home-assistant.io/cookbook/) are generated automatically, so there is no need to add a link to those pages.
+To create a new integration page, follow these steps:
 
-Please honor the [Standards](documenting/standards.md) we have for the documentation.
+1. The fastest way is to make a copy of the [integration documentation template](https://github.com/home-assistant/home-assistant.io/tree/current/source/_integrations/_integration_docs_template.markdown) and edit it.
+2. Make sure the filename of the integration page matches the domain name of the integration.
+3. The [Integration overview](https://www.home-assistant.io/integrations/) and the [Examples section](https://www.home-assistant.io/cookbook/) are generated automatically, so there is no need to add a link to those pages.
```

Comment:
this one is a bit confusing as a "step". maybe make it a sub bullet point on step 1, or as a note outside the steps?